docker/min_settings.py

```diff
@@ -56,9 +56,9 @@
     'host': redis_host,
 }
 
-ELASTICSEARCH_HOST = 'elasticsearch5'
-ELASTICSEARCH_PORT = 9200  # ES 5 port
-ELASTICSEARCH_MAJOR_VERSION = 5
+ELASTICSEARCH_HOST = 'elasticsearch6'
+ELASTICSEARCH_PORT = 9200  # ES 6 port
```

Comment:
nit: does this comment actually provide any value? Even if this port did change across versions, I'd assume we would need to have some logic in this setting to determine which port to use. The name seems clear enough to me, and at this point the comment just adds another thing to have to maintain over time.

Reply:
https://github.com/dimagi/commcare-hq/pull/36468/commits/7f08893e061206b4fdce96d6339bd22c3a6672b2